src/main/java/bio/terra/cli/command/workspace/Clone.java

```diff
@@ -21,6 +21,9 @@
 /** This corresponds to the third-level "terra workspace clone" command. */
 @Command(name = "clone", description = "Clone an existing workspace.")
 public class Clone extends BaseCommand {
+  @CommandLine.Option(names = "--id", required = true, description = "Workspace ID")
```

Comment:
Indicate if this is for the source or destination workspace.

Reply:
done